rules-tests/Transform/Rector/MethodCall/MethodCallToMethodCallRector/Fixture/dont_replace_funcall.php.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Rector\Tests\Transform\Rector\MethodCall\MethodCallToMethodCallRector\Fixture;
+
+use Rector\Tests\Transform\Rector\MethodCall\MethodCallToMethodCallRector\Source\FirstDependency;
+
+function aFunctionReturningFirstDependency() : FirstDependency {
+    return new FirstDependency();
+}
+
+aFunctionReturningFirstDependency()->go();
+?>
+-----
+<?php
+
+namespace Rector\Tests\Transform\Rector\MethodCall\MethodCallToMethodCallRector\Fixture;
+
+use Rector\Tests\Transform\Rector\MethodCall\MethodCallToMethodCallRector\Source\FirstDependency;
+
+function aFunctionReturningFirstDependency() : FirstDependency {
+    return new FirstDependency();
+}
+
+aFunctionReturningFirstDependency()->go();
+?>
```

Comment:
same code no need to have `after` change, so start from `----` can be removed.

```suggestion
```

also, please change fixture with `skip_replace_func_call.php.inc` for consistency.

Reply:
please create also test fixture with inside class to avoid create property from `FuncCall`

Reply:
Oh, nice tips.

I'll make the name change.

I wanted to discuss that test with you by the way. Is it a good test case for https://github.com/rectorphp/rector-src/pull/2554#discussion_r904038907 ?

It passed directly so maybe you had something else in mind.

Reply:
`FuncCall` inside class fixture is needed.

Reply:
Sorry, I'm not sure I understand what you mean here.

Do you mean a test fixture like this 

```php
<?php

namespace Rector\Tests\Transform\Rector\MethodCall\MethodCallToMethodCallRector\Fixture;

use Rector\Tests\Transform\Rector\MethodCall\MethodCallToMethodCallRector\Source\FirstDependency;

function aFunctionReturningFirstDependency() : FirstDependency {
    return new FirstDependency();
}

class AClass {
    public function aMethod()
    {
        aFunctionReturningFirstDependency()->go();
    }
}

?>

```

Reply:
Yes

Reply:
This one breaks the tests :)

Reply:
That's why the `PropertyFetch` or `Variable` check is needed :)